Wondermovie updates

MeuhV1 opened the pull request:
Fixed double wondermovies for projects
Wondermovies only appear the first time a buildings/wonder is build
Removed condition that movie only appear for world wonders

Commit:
Fixed double wondermovies for projects
Wondermovies only appear the first time a buildings/wonder is build

dguenms commented:
What are the double project movies about? Are the movies also triggered by the DLL so they appear twice when the Python code is also executed?

MeuhV1 commented:
I don't think the DLL has anything to do with it. The python code that triggers it was coded twice, so I removed one trigger. This is probably introduced when merging platybuilder. Wonders did have the same problem, but I fixed that already in a previous pull request.

dguenms commented:
Oh it's from the Platybuilder merge, got it. Merged.
